### PR TITLE
Object.assign() opressor added

### PR DESCRIPTION
**Pull request by Nyalab**

### CODE REVIEW

**Comment by MoOx** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -38,6 +38,23 @@ selector.on("click", function() {
...
+
+You cannot morally grant yourself the right to assign something on things.
+
+```javascript
+// bad
+var newObject = Object.assign(prevObject, nextProps)
+
+// good
+for(var key in prevObject) {
+  // this step is important, always ask for permission ;)
```
Comment (bloodyowl) [inline README.md]: stop oppressing one-eyed people

File: README.md [hunk 2]
```diff
@@ -38,6 +38,23 @@ selector.on("click", function() {
...
+  if(prevObject.hasOwnProperty(key)) {
+    newObject[key] = prevObject[key]
```
Comment (bloodyowl) [inline README.md]: where does `newObject` come from?

**Comment by Nyalab** [comment]
:nazi:

**Pull request merged**